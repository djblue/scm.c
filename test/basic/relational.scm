(write (< 1))
(write (< 1 2))
(write (< 1 2 3))
(write (< 1 1))
(write (< 1 0))
(write (< 1 1 1))

(write (<= 1))
(write (<= 1 2))
(write (<= 1 2 3))
(write (<= 1 1))
(write (<= 1 0))
(write (<= 1 1 1))

(write (> 1))
(write (> 2 1))
(write (> 3 2 1))
(write (> 1 1))
(write (> 0 1))
(write (> 1 1 1))

(write (>= 1))
(write (>= 2 1))
(write (>= 3 2 1))
(write (>= 1 1))
(write (>= 0 1))
(write (>= 1 1 1))
