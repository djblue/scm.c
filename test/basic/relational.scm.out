#t
#t
#t
#f
#f
#f
#t
#t
#t
#t
#f
#t
#t
#t
#t
#f
#f
#f
#t
#t
#t
#t
#f
#t
